### ISSUE TITLE
[a11y][ActionList] `:active` state not visible in WCHM

### ISSUE THREAD

**Issue by emma-boardman** [Shopify/polaris-react#3717]
## Issue summary

The active/selected indicator for `ActionList` > `Item` is not visible in Windows High Contrast Mode (Edge, Windows VM)

## Expected behavior

- Visual distinction between `active` and inactive items (the indigo vertical line to the left of the option) to be visible in WCHM 

![image](https://user-images.githubusercontent.com/17032120/102211313-56c58300-3ecb-11eb-9f5a-d458f1c20de5.png)
 

## Actual behavior

- No distinction between `active` and inactive items. 

![image](https://user-images.githubusercontent.com/17032120/102211443-84aac780-3ecb-11eb-8e33-db15e2519542.png)

## Steps to reproduce the problem

1. Access the [reduced test case CodeSandbox](https://codesandbox.io/s/actionlist-wchm-reduced-test-case-gbgpn?file=/App.js) in Edge, on a Windows device or VM 
2. Turn on High Contrast Mode
3. Open Popover and observe visual state of `import` (active), and `export` (not active). 

## Reduced test case

https://codesandbox.io/s/actionlist-wchm-reduced-test-case-gbgpn?file=/App.js

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 5.12.0 
- Browser: Edge 87.0.664.47 on Windows 10
- Device: Windows 10 on VirtualBox 6 VM 
- Operating System: Windows 10 (on MacOS Window VM)
